app/models/garden.rb

```diff
@@ -0,0 +1,6 @@
+class Garden < ApplicationRecord
+  belongs_to :user
+  has_many :plants
+  validates_presence_of :name, :zip_code
+  accepts_nested_attributes_for :plants, reject_if: :all_blank
```

Comment:
Oh, I like the `reject_if` command!

Reply:
These nested attributes and reject_if are so fancy and cool

Reply:
I know right? I was worried it might not work so well for individual plant create, but it seems to work just fine!